### PR TITLE
[WIP] Updated descriptions for furniture and terrain

### LINKED ISSUE #28497: Add extended description for more non-obvious tiles
**Is your feature request related to a problem? Please describe.**  
There are some tiles in the game that are not too self-explanatory and would thus benefit from an extended description. Many tiles are still lacking one, or the current extended description isn't that good so far.

**Describe the solution you'd like**  
Write an extended description for most tiles and describe how the “interactive” tiles can be used. Some tiles are a bit more complicated and are only useful in combination with other ti

The description should reflect:

* For interactive tiles: How they work and how they can be used (most important). Also, if it applies: how they interact with other tiles (example: fuel console)
* For machines and apparatuses that can't be used (like the arcade machine): Say in description that these are “broken”. Also maybe say what their pre-Cataclysm use was, in 1 sentence
* For furniture: (roughly) what they are made of (implications for deconstruction)
* For all tiles: What they are
* For all tiles: Whatever you write, write it spoiler-free

**High priority tiles**
- t_cvdmachine
- t_atm
- t_slot_machine (ext. description should explain the game, with all possible winnings and chances)
- t_gas_pump*
- t_diesel_pump*
- t_radio_controls
- t_marloss*
- t_radio_tower
- t_console
- t_gates_control*
- t_centrifuge
- t_manhole*
- t_card
- t_plut_generator

Tiles that might need an improvement:

* mailbox (“You've got mail.” is false)
* f_arcade_machine (cannot be used)
* f_pinball_machine (cannot be used)
* ball machine (cannot be used)
**Describe alternatives you've considered**  
None.

**Additional context**  
None.

### PR DESCRIPTION
**Pull request by Pr0man**
#### Summary
SUMMARY: Content "Added descriptions to furniture and and terrain."

#### Purpose of change
Fixes #28497 
Added several extended descriptions for furniture and terrain.

#### Describe the solution
Many of the items were lacking a `description` field alltogether. By describing what a tile looks like without going into too much detail about its use, immersion for those willing to read them can be increased while also providing hints for those stunted.

#### Describe alternatives you've considered
None, this will most likely have to be done at one point or another.

#### Additional context
Not entirely sure as to how to handle the more joking or quirky descriptions for furniture like chairs or mirrors. While they do add a certain charm with their approach, they shouldn't be overdone. Nevertheless, expanding the desciptions of quirky text string to not just include a joke but also a description beforehand might work as a compromise.

### CODE REVIEW

**Comment by Ilysen** [comment]
I've already added a few descriptions in #28292, but it's on hold until 0.D is merged. I'd humbly request you make sure your additions don't conflict with those!

**Comment by Pr0man** [comment]
> 
> 
> I've already added a few descriptions in #28292, but it's on hold until 0.D is merged. I'd humbly request you make sure your additions don't conflict with those!

ohhh good stuff - once I pick up work I'll remove any entries made that would conflict with yours and just add more descriptors to objects and furniture

File: data/json/terrain.json [hunk 1]
```diff
@@ -6697,6 +6714,7 @@
     "type": "terrain",
     "id": "t_gates_control_metal_lab",
     "name": "control lever",
+    "description": "A small two-position lever which allows you to either close or open a gate.  Due to it being inside a laboraty it most likely is electronic and connected to the grid.",
```
Comment (anothersimulacrum) [inline data/json/terrain.json]: laboraty > laboratory

File: data/json/terrain.json [hunk 2]
```diff
@@ -6990,6 +7011,7 @@
     "type": "terrain",
     "id": "t_manhole",
     "name": "manhole",
+    "description": "A tight tunnel with a metal steps interlaced into the wall allowing for (relatively) easy access downwards.  You could potentially squeeze through and climb down but do you really want to?",
```
Comment (anothersimulacrum) [inline data/json/terrain.json]: This would sound better with a comma after climb down

File: data/json/terrain.json [hunk 3]
```diff
@@ -7117,6 +7143,7 @@
     "type": "terrain",
     "id": "t_slot_machine",
     "name": "slot machine",
+    "description": "A new electronic slot machine that now only collects dust instead of coins since it has no power.  On the sides to it you can still see the different payouts ranging from a mere $5 to over $500 dollars available at checkout.",
```
Comment (anothersimulacrum) [inline data/json/terrain.json]: `$500 dollars` is redundant

File: data/json/terrain.json [hunk 4]
```diff
@@ -6344,6 +6350,7 @@
     "name": "ATM",
     "symbol": "&",
     "color": "magenta",
+    "description": "A standard automatic teller machine that is still connected to power and firmly planted into the ground.  You could probably still use it to transfer money around though that may have little use in the cataclysm.",
```
Comment (anothersimulacrum) [inline data/json/terrain.json]: Would flow better with a comma after around

File: data/json/terrain.json [hunk 5]
```diff
@@ -6528,6 +6537,7 @@
     "type": "terrain",
     "id": "t_console",
     "name": "computer console",
+    "description": "A console with a brightly lit, green-monochrome screen.  You might be able to access its content provided that you know what you are doing.",
```
Comment (anothersimulacrum) [inline data/json/terrain.json]: content > contents

**Comment by ZhilkinSerg** [comment]
Marking as stalled. Please ping me to reopen when you are ready to pick up where you left.

**Pull request closed without merging**